### PR TITLE
[15.0][MIG] project_wbs_stage

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Adapt v15 base project stage model

### CODE REVIEW

File: project_wbs_stage/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline project_wbs_stage/__manifest__.py]:
```suggestion
# Copyright 2017-23 ForgeFlow S.L.
```

File: project_wbs_stage/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017-23 ForgeFlow S.L.
+# Copyright 2017-19 Luxim d.o.o.
+# Copyright 2017-19 Matmoz d.o.o.
+# Copyright 2017-19 Deneroteam.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Project WBS Stage",
+    "version": "14.0.1.0.0",
```
Comment (JuanyDForgeflow) [inline project_wbs_stage/__manifest__.py]: Change it to "15.0.1.0.0"

**Review by JuanyDForgeflow — APPROVED** [review]
LGTM, precommit works locally it seems like the error is unrelated to the module 
image

**Comment by AaronHForgeFlow** [comment]
Just rebased to see if pre-commit is fixed

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**